### ISSUE TITLE
Autodiscover overrides manually added servers

### ISSUE THREAD

**Issue by mcarlton00** [jellyfin/jellyfin-kodi#63]
**Note:** As far as I can tell, this _only_ happens when running multiple docker instances on the same server, and autodiscover (`--net=host`) is enabled on one of them.

While trying to do some debugging on a new testing instance, I'm unable to login to the server.  It appears to be trying to get an authentication token from the instance with autodiscover enabled, instead of the one I've manually defined.

The setup:
Instance 1 - `jellyfin-prod`.  Created using `--net=host`
instance 2 - `jellyfin-testing`.  Created with `--publish 8097:8097`, and the port changed within Jellyfin.  Authentication through the browser directly to `http://IP:8097` and through a nginx reverse proxy works as expected, but fails from jf-kodi.

Results:
It connects to the host that I type into the manually add box
```
tryConnect url: http://192.168.0.200:8097/emby/system/info/public
```
This gives the correct server name and user list on the next 2 panels.

However, after selecting a user and trying to login:
```
Begin connectToServers, with 1 servers
begin connectToServer
beginning connection tests
testing connection mode 0 with server Zeus
tryConnect url: http://192.168.0.200:8096/emby/system/info/public
calling onSuccessfulConnection with connection mode 0 with server Zeus
```
it's trying to connect to the 8096 instance instead.

**Comment by oddstr13** [comment]
Could be related to this madness?

https://github.com/jellyfin/jellyfin-kodi/blob/e92d60f7c70d16069602780a8a92a59eef9260e9/resources/lib/jellyfin/core/connection_manager.py#L456L470

I'm not entirely sure of what that code is trying to do.

**Comment by TrueTechy** [comment]
I noticed the opposite of this recently. If you manually add a server the autodiscovered ones are gone from the list

**Comment by mcarlton00** [comment]
Seems like this was fixed in #307
